travvy/src/components/LogIn.vue

```diff
@@ -40,22 +42,36 @@
       //the login method, this is what runs when the log in button is pushed
       async login(){
         try {
-          //call the login method from AuthenticationService
-          //email and password are parameters
-          const isLoggedIn = await AuthenticationService.login({
-            "email": this.email,
-            "password": this.password
-          })
-          console.log(isLoggedIn)
-          //if the login is valid
-          if (isLoggedIn){
-            //store the user's email
-            this.$store.dispatch('setUserEmail', this.email)
-            //go to the home page
-            this.$router.push("Home")
-          }else{
-            //if the login is not valid, do nothing and tell the user it was wrong
-            alert("The log in information was incorrect, please try again")
+          //create a regex function for valid email
+          var re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+          //test the email
+          const validEmail= re.test(this.email);
```

Comment:
Glad to see you were able to implement RegEx this sprint! I found it very useful in sprint 3.